x/authz/client/cli/query.go

```diff
@@ -96,13 +96,13 @@ $ %s query %s grants cosmos1skjw.. cosmos1skjwj.. %s
 
 func GetQueryGranterGrants() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "granter-grants [granter-addr]",
+		Use:   "grants-by-granter [granter-addr]",
```

Comment:
@atheeshp Do you know if there's a way to create cli command aliases?

I would like to ideally not introduce a CLI-breaking change

Reply:
Cobra supports that https://github.com/spf13/cobra/blob/master/command.go#L51 :)